src/domain/users-ips/index.types.d.ts

```diff
@@ -3,6 +3,17 @@ type UserIPs = {
   lastIPs: IPType[]
 }
 
+type IPMetadataValidatorArgs = {
+  denyIPCountries: string[]
+  allowIPCountries: string[]
+  denyASNs: string[]
+  allowASNs: string[]
+}
+
+type IPMetadataValidator = {
+  validateForReward(ipMetadata?: IPType): true | ApplicationError
```

Comment:
`ValidationError` here instead of `ApplicationError`?

Reply:
Fixed and `PhoneMetadataValidator` too https://github.com/GaloyMoney/galoy/pull/1232/commits/9333231b2b3e87f0f2eb04110a8f03bcee7ac70b